src/core/border_router/routing_manager.hpp

```diff
@@ -1535,14 +1539,16 @@ class RoutingManager : public InstanceLocator
         void EvaluateStateChange(State aOldState);
         void WithdrawPrefix(void);
         void StartStop(bool aStart);
+        void PauseResume(bool aPause);
 
         static const char *StateToString(State aState);
 
         using PrefixTimer   = TimerMilliIn<RoutingManager, &RoutingManager::HandlePdPrefixManagerTimer>;
         using StateCallback = Callback<PdCallback>;
 
         bool          mEnabled;
-        bool          mIsRunning;
+        bool          mIsStarted;
+        bool          mIsPaused;
```

Comment:
Maybe in a future PR. I think it'd be good to see if we can combine all of these booleans into a  single `mState`.

Reply:
Will track in #10741 10741